gui-apps/swaylock-effects/swaylock-effects-1.6.4.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 2021 Aisha Tammy, Ichika Zou
+# Distributed under the terms of the ISC License
+
+EAPI=8
+
+inherit bash-completion-r1 meson
+
+DESCRIPTION="Screen locker for Wayland"
+HOMEPAGE="https://github.com/mortie/swaylock-effects"
+
+MY_PV=$(ver_rs 2 '-')
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/mortie/swaylock-effects.git"
+else
+	SRC_URI="https://github.com/mortie/swaylock-effects/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}"/${PN}-${MY_PV}
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="fish-completion +gdk-pixbuf pam cpu_flags_x86_sse"
```

Comment:
I did some poking around and seems like fish completions doesn't actually need fish to be installed. So I think we can remove this flag and install the completions by default.